core/kazoo_apps/src/kapps_account_config.erl

```diff
@@ -300,7 +300,7 @@ get(Account, Category) ->
         {'error', _} -> kz_doc:set_id(kz_json:new(), kapps_config_util:account_doc_id(Category))
     end.
 
--spec load_config_from_system(kz_term:api_binary(), kz_term:ne_binary()) -> kazoo_data:get_results_return().
+-spec load_config_from_system(kz_term:api_binary(), kz_term:ne_binary()) -> kazoo_data:jobj_return().
```

Comment:
`get_results_return()` allows for JSON terms (like arrays and whatnot) which are valid results from views. It is not a given that a view request will return just objects.